### ISSUE TITLE
Javascript error blocks dataviz rendering (staging)

### ISSUE THREAD

**Issue by jde** [bocoup/coral-ask-election-2016#83]
When the form loads, a javascript error is thrown reading the data packet:

```
selectors.js:115Uncaught (in promise) TypeError: Cannot read property 'count' of undefined(…)
```

This bug is visible here:
https://s3-us-west-2.amazonaws.com/ask-stg/build/index.html

Datafile:
https://s3-us-west-2.amazonaws.com/ask-stg/form-5818fffcba6bce0001ba8ab8-aggregation-digest.json

**Comment by kadamwhite** [comment]
@jde please pull latest and re-test.

**Comment by kadamwhite** [comment]
Closed by #82
